client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.ts

```diff
@@ -314,16 +322,20 @@ export class ParticipantDetailViewComponent extends BaseMeetingComponent {
      * @param edit
      */
     public async setEditMode(edit: boolean): Promise<void> {
-        if (!this.newUser && edit) {
-            this._isUserInScope = await this.userService.hasScopeManagePerms(this._userId!);
-        }
-
         this.isEditingSubject.next(edit);
 
         // case: abort creation of a new user
         if (this.newUser && !edit) {
             this.goToAllUsers();
         }
+
+        if (edit) {
+            setTimeout(() => {
+                this._userFormLoaded = true;
+            }, 1000);
```

Comment:
Use an actual check if the data is loaded.

Reply:
I added this workaround, because the save button was flickering.
The save button is enabled/disabled because the user-detail-view validEvent fires 4 times in the buildup of the
form. Mmh, perhaps I find the problem here.